substrate/frame/conviction-voting/src/lib.rs

```diff
@@ -107,9 +105,8 @@ pub mod pallet {
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
 		/// Currency type with which voting happens.
-		type Currency: ReservableCurrency<Self::AccountId>
-			+ LockableCurrency<Self::AccountId, Moment = BlockNumberFor<Self>>
-			+ fungible::Inspect<Self::AccountId>;
+		type Currency: MutateHold<Self::AccountId, Reason = Self::RuntimeHoldReason>
```

Comment:
and the comment above need to be updated

```suggestion
		type Fungible: MutateHold<Self::AccountId, Reason = Self::RuntimeHoldReason>
```

Reply:
I would keep `Currency` here, this has already been kept in other migrations like
https://github.com/paritytech/substrate/pull/14020/files#diff-70e9723e9db62816e35f6f885b6770a8449c75a6c2733e9fa7a245fe52c4656cR210-R213

There was a support from @shawntabrizi and @liamaharon towards renaming in here https://github.com/paritytech/substrate/pull/14020#issuecomment-1652250249, but it didn't end happening.

My reasoning behind not renaming is that, at the end of the day, the type is still representing a currency, even though it does not implement a currency trait directly but rather a summary of other traits that put together do represent a currency in this context

Reply:
Honestly I think it's just confusing and we should rename them all.

> the type is still representing a currency
The type also still represents a fungible. and it's much more descriptive, clear, and less prone to being misinterpreted to call it Fungible.